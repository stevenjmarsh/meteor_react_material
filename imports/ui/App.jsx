/* eslint-disable arrow-parens, object-curly-newline */
import React, { Component, Fragment } from "react";
import Header from "./components/Header";

// App component - represents the whole app
export default class App extends Component {
  state = {};

  render() {
    return (
      <Fragment>
        <Header />
      </Fragment>
    );
  }
}
