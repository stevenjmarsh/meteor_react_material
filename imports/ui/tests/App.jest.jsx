/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import { Typography } from "@material-ui/core";
import App from "../App";

describe("<App />", function() {
  it("matches render snapshot", function() {
    const tree = renderer.create(<App />);
    expect(tree).toMatchSnapshot();
  });

  it("contains a Typography component with 'Hello!'", function() {
    const wrapper = shallow(<App />);
    expect(wrapper.find(Typography).prop("children")).toEqual("Hello!");
  });
});
