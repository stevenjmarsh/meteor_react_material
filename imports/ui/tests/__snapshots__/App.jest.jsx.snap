// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches render snapshot 1`] = `
<div
  className="Header-root-1"
>
  <header
    className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorPrimary-9"
  >
    <div
      className="MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39"
    >
      <h6
        className="MuiTypography-root-42 MuiTypography-h6-59 MuiTypography-colorInherit-71"
      >
        Hello!
      </h6>
    </div>
  </header>
</div>
`;
